drivers/counter/counter_qmsi_aonpt.c

```diff
@@ -108,8 +109,7 @@ static u32_t aon_timer_qmsi_read(struct device *dev)
 }
 
 static int aon_timer_qmsi_set_alarm(struct device *dev,
```

Comment:
legacy set_alarm function has been changed to set_wrap_alarm so this function should actually implement interface

```
typedef int (*counter_api_set_wrap_alarm)(struct device *dev,
					  u32_t ticks,
					  counter_wrap_callback_t callback,
					  void *user_data);

set_alarm should be used if device has more compare channels and alarms are set within counter period (which is set by set_wrap_alarm).
```

Reply:
ok . I change it.